grpcdogfood/chaos-demo-namespace.yaml

```diff
@@ -0,0 +1,9 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Why can't we re-use manifests in the `/examples/demo.yaml` folder which already contains this namespace instead of duplicating it?

Reply:
True, what do you think of the update? I broke out the namespace so it can be independently applied of the original demo curl pods (which are not needed for gRPC disruption), then included in the main contributing instructions that you need to instantiate it separately. This may disrupt colleague workflows, so it's also possible to just create a new namespace for gRPC demos and keep the manifests separate.